src/components/TextArea/TextArea.css

```diff
@@ -14,6 +14,7 @@
   display: flex;
   font-family: var(--textarea-font-family);
   margin: var(--textarea-margin, 0);
+  overflow: hidden;
```

Comment:
if the text area will keep on growing why do we need to set overflow?

Reply:
@kaiomagalhaes, as you can see on the image below if we don't set the overflow the scrollbar shows up even if the input value is empty.

![image](https://user-images.githubusercontent.com/40361370/148223136-de90d829-cc75-4a1a-83ea-6f921a6fb6ab.png)